### ISSUE TITLE
Add Discord link on the main menu

### ISSUE THREAD

**Issue by GitSparTV** [Facepunch/garrysmod-requests#1682]
Unlike Rust, GMod doesn't have any link to interact with the community except the workshop and news at the top right corner.
I think Discord and/or other official community links should be on the main menu to connect people.

**Comment by nodgear** [comment]
Better than that, i think discord link should be embedded on gmod blog.

**Comment by GitSparTV** [comment]
Piggo had additional idea to add Artworks section.

I think the general idea is to add more connections to the menu, not just Discord

**Comment by GitSparTV** [comment]
@robotboy655 You recently changed main menu, what do you think about this request?

**Comment by robotboy655** [comment]
I don't see a good place for the button in the main menu

**Comment by GitSparTV** [comment]
Where "problems" is maybe?
Docked left or right.

Or near news, similar to rust

**Comment by robotboy655** [comment]
I don't think it fits on the bottom navbar as it has no relevance to everything else on it.

**Comment by GitSparTV** [comment]
What if near news?

**Comment by GitSparTV** [comment]
Small icons for discord, wiki and maybe something else
